source/simulator/checkpoint_restart.cc

```diff
@@ -392,6 +392,16 @@ namespace aspect
 #if DEAL_II_VERSION_GTE(9,1,0)
             move_file (parameters.output_directory + "restart.mesh_fixed.data",
                        parameters.output_directory + "restart.mesh_fixed.data.old");
+
+            std::fstream check_variable_file((parameters.output_directory + "restart.mesh_variable.data").c_str());
+            const bool variable_data_file_exists = check_variable_file.is_open();
```

Comment:
can you use ``Utilities::fexists()`` instead?

Reply:
Absolutely, forgot about that function.